### PR TITLE
Project actions

### PR DESCRIPTION
**Pull request by aginzberg**
Removes project level from my projects. Allows pausing/resuming, editing/deleting, and forking of projects. These actions are only allowable depending on the status of the project.

Note: when md-secondary comes out we can make the icons overlay the highlighted list item better and avoid all of the stopPropagation junk

### CODE REVIEW

**Comment by dmorina** [comment]
grrrr ... need to update sql migrations to status=2

**Comment by aginzberg** [comment]
i dont think so. status 2 is used only temporarily when the status is first being published, when the tasks need to be created for the first time. status 3 is when the project is in progress and should be displayable on the task feed.

**Comment by dmorina** [comment]
clicking on the buttons (pause/fork) opens the module on Firefox

File: static/js/project/controllers/my-project.controller.js [hunk 1]
```diff
@@ -71,8 +77,77 @@
...
+            Project.update(item.id, {status: 3}, 'module').then(
+                function success(response) {
+                    $mdToast.showSimple('Resumed ' + item.name + '!');
+                    item.status = 3;
+                },
+                function error(response) {
+                    $mdToast.showSimple('Could not resume module.');
+                }
+            ).finally(function () {});
+        }
```
Comment (dmorina) [inline static/js/project/controllers/my-project.controller.js]: please merge resume and pause into one function

**Comment by dmorina** [comment]
lgtm

**Pull request merged**